rPackages.prqlr: fixed build

b-rodrigues opened the pull request:
Fixed build, fix works for current version, and also for the next version in the upcoming bump.

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

HannahMR committed:
lightning-terminal: init at 0.14.1-alpha

Jegp committed:
morgen: 3.5.9 -> 3.6.6

FliegendeWurst committed:
perlPackages.Tirex: fix build with recent mapnik

rnhmjoj committed:
nixos/grub: generate BLS entries

rnhmjoj committed:
nixos/tests/grub: check boot loader entries

rnhmjoj committed:
nixos/tests/nixos-rebuild-install-bootloader: check boot loader entries

rnhmjoj committed:
nixos/release-notes: mention GRUB boot loader entries

matdibu committed:
rasdaemon: 0.8.0 -> 0.8.2

matdibu committed:
nixos/rasdaemon: add package option

r-ryantm committed:
spdx-license-list-data: 3.25.0 -> 3.26.0

FliegendeWurst committed:
perlPackages.SDL: fix build with GCC 14

FKouhai committed:
hubble: 0.13.6 -> 1.17.1

phanirithvij committed:
a-keys-path: init at 0.7.1

Signed-off-by: phanirithvij <phanirithvij2000@gmail.com>
Co-authored-by: Gaétan Lepage <33058747+GaetanLepage@users.noreply.github.com>

kpbaks committed:
maintainers: add kpbaks

kpbaks committed:
simple-completion-language-server: init at version 0-unstable-2025-01-31

pyrox0 committed:
webpack-cli: 5.1.4 -> 6.0.1

FKouhai committed:
maintainers: add FKouhai

fpletz committed:
bind.dnsutils: remove reference to main output

The `delv` executable references `$out/etc/bind.keys` but a copy of that
file is included in binary anyway. DNSSEC validation still works when
the reference is removed.

Pleune committed:
mxnet: remove cuda support

mxnet is unmaintained, and requires cuda<=11 to compile. Removed
support completely as most projects are migrating away from mxnet, and
nixpkgs will soon stop supporting these old cuda versions.

r-ryantm committed:
awscli2: 2.23.11 -> 2.24.7

FredeHoey committed:
maintainers: update fredeb

cloudripper committed:
goose-cli: 1.0.7 -> 1.0.10, add cloudripper as maintainer, remove nayeko as maintainer

fgaz committed:
faircamp: 1.1.1 -> 1.2.0

Diff: https://codeberg.org/simonrepp/faircamp/compare/1.1.1...1.2.0

wegank committed:
freefilesync: 14.0 -> 14.2

r-ryantm committed:
sql-formatter: 15.4.10 -> 15.4.11

genga898 committed:
aonsoku: init at 0.8.3

aonsoku: add updateScript and mark darwin as broken

aonsoku: remove badPlatforms

aonsoku: use finalAttrs

felbinger committed:
python313Packages.drf-extra-fields: add pytz dependency

Levizor committed:
tray-tui: init at 0.1.0

chvp committed:
mu: 1.12.8 -> 1.12.9

https://github.com/djcb/mu/releases/tag/v1.12.9

r-ryantm committed:
python312Packages.pycuda: 2024.1.2 -> 2025.1

jbedo commented on pkgs/development/r-modules/default.nix:
```diff
@@ -1227,6 +1227,11 @@ let
     });
 
+   prqlr = old.prqlr.overrideAttrs (attrs: {
+      nativeBuildInputs = with pkgs; [ cargo rustc ] ++ attrs.nativeBuildInputs;
```
Can you move this to `packagesWithNativeBuildInputs`?

r-ryantm committed:
c-blosc2: 2.16.0 -> 2.17.0

GaetanLepage committed:
python312Packages.flax: 0.10.3 -> 0.10.4

Diff: https://github.com/google/flax/compare/refs/tags/v0.10.3...v0.10.4

Changelog: https://github.com/google/flax/releases/tag/v0.10.4

usertam committed:
pcl: fix build on darwin

gesperon committed:
lenovo-legion-app: 0.0.18 -> unstable-2025-02-20

SuperSandro2000 committed:
python313Packages.ipwhois: use libredirect.hook

SuperSandro2000 committed:
fwupd: use libredirect.hook

r-ryantm committed:
minio-warp: 1.1.0 -> 1.1.1

fabaff committed:
python313Packages.accuweather: 4.0.0 -> 4.1.0

Diff: https://github.com/bieniu/accuweather/compare/refs/tags/4.0.0...4.1.0

Changelog: https://github.com/bieniu/accuweather/releases/tag/4.1.0

fabaff committed:
python313Packages.aiohomeconnect: 0.13.0 -> 0.15.1

Diff: https://github.com/MartinHjelmare/aiohomeconnect/compare/refs/tags/v0.13.0...v0.15.1

Changelog: https://github.com/MartinHjelmare/aiohomeconnect/blob/refs/tags/v0.15.1/CHANGELOG.md

fabaff committed:
python313Packages.aiolifx-themes: 0.6.7 -> 0.6.8

Diff: https://github.com/Djelibeybi/aiolifx-themes/compare/refs/tags/v0.6.7...v0.6.8

Changelog: https://github.com/Djelibeybi/aiolifx-themes/releases/tag/v0.6.8

fabaff committed:
python313Packages.aiostreammagic: 2.10.0 -> 2.11.0

Diff: https://github.com/noahhusby/aiostreammagic/compare/refs/tags/2.10.0...2.11.0

Changelog: https://github.com/noahhusby/aiostreammagic/releases/tag/2.11.0

fabaff committed:
python313Packages.aiounifi: 81 -> 83

Diff: https://github.com/Kane610/aiounifi/compare/refs/tags/v81...v83

Changelog: https://github.com/Kane610/aiounifi/releases/tag/v83

fabaff committed:
python312Packages.mypy-boto3-batch: 1.37.0 -> 1.37.2

fabaff committed:
python312Packages.mypy-boto3-chime: 1.37.0 -> 1.37.2

fabaff committed:
python312Packages.mypy-boto3-cloudfront: 1.37.0 -> 1.37.2

fabaff committed:
python312Packages.mypy-boto3-ec2: 1.37.1 -> 1.37.2

fabaff committed:
python312Packages.mypy-boto3-emr: 1.37.0 -> 1.37.3

fabaff committed:
python312Packages.mypy-boto3-iotfleetwise: 1.37.0 -> 1.37.2

fabaff committed:
python312Packages.mypy-boto3-oam: 1.37.0 -> 1.37.2

fabaff committed:
python312Packages.mypy-boto3-redshift-serverless: 1.37.0 -> 1.37.3

fabaff committed:
python312Packages.mypy-boto3-sagemaker: 1.37.0 -> 1.37.3

fabaff committed:
python312Packages.mypy-boto3-storagegateway: 1.37.0 -> 1.37.3

r-ryantm committed:
bosh-cli: 7.9.2 -> 7.9.3

r-ryantm committed:
python312Packages.google-cloud-dataproc: 5.17.0 -> 5.18.0

r-ryantm committed:
python312Packages.drf-yasg: 1.21.8 -> 1.21.9

r-ryantm committed:
dapr-cli: 1.14.1 -> 1.15.0

r-ryantm committed:
docker-credential-helpers: 0.8.2 -> 0.9.0

r-ryantm committed:
gitlab-ci-ls: 1.0.1 -> 1.0.2

r-ryantm committed:
infrastructure-agent: 1.60.1 -> 1.62.0

r-ryantm committed:
prometheus-artifactory-exporter: 1.15.0 -> 1.15.1

r-ryantm committed:
src-cli: 6.0.1 -> 6.1.0

r-ryantm committed:
stylance-cli: 0.5.4 -> 0.5.5

r-ryantm committed:
tfswitch: 1.3.0 -> 1.4.0

xiaoxiangmoe committed:
taze: init at 18.6.0

xiaoxiangmoe committed:
bumpp: init at 10.0.3

r-ryantm committed:
python312Packages.google-cloud-dlp: 3.27.0 -> 3.28.0

wrbbz committed:
pulumi-bin: 3.152.0 -> 3.153.0

r-ryantm committed:
gqlgen: 0.17.65 -> 0.17.66

wegank committed:
msolve: 0.7.3 -> 0.7.4

r-ryantm committed:
candy-icons: 0-unstable-2025-02-08 -> 0-unstable-2025-02-23

r-ryantm committed:
summon: 0.10.2 -> 0.10.3

r-ryantm committed:
python312Packages.lm-format-enforcer: 0.10.9 -> 0.10.11

wegank committed:
msolve: improve darwin support

rorosen committed:
k3s_1_29: 1.29.13 -> 1.29.14

https://github.com/k3s-io/k3s/releases/tag/v1.29.14%2Bk3s1

rorosen committed:
k3s_1_30: 1.30.9 -> 1.30.10

https://github.com/k3s-io/k3s/releases/tag/v1.30.10%2Bk3s1

rorosen committed:
k3s_1_31: 1.31.5 -> 1.31.6

https://github.com/k3s-io/k3s/releases/tag/v1.31.6%2Bk3s1

rorosen committed:
k3s_1_32: 1.32.1 -> 1.32.2

https://github.com/k3s-io/k3s/releases/tag/v1.32.2%2Bk3s1

r-ryantm committed:
python312Packages.publicsuffixlist: 1.0.2.20250227 -> 1.0.2.20250228

GaetanLepage committed:
ruff: 0.9.8 -> 0.9.9

Diff: https://github.com/astral-sh/ruff/compare/refs/tags/0.9.8...0.9.9

Changelog: https://github.com/astral-sh/ruff/releases/tag/0.9.9

erictapen committed:
weblate: 5.10.1 -> 5.10.2

Changelog: https://github.com/WeblateOrg/weblate/releases/tag/weblate-5.10.2

r-ryantm committed:
amnezia-vpn: 4.8.3.2 -> 4.8.4.1

r-ryantm committed:
python312Packages.python-roborock: 2.11.2 -> 2.12.0

zowoq committed:
rasdaemon: 0.8.0 -> 0.8.2 (#369375)

r-ryantm committed:
pretender: 1.3.1 -> 1.3.2

GaetanLepage committed:
beetsPackages.alternatives: 0.13.1 -> 0.13.2

Diff: https://github.com/geigerzaehler/beets-alternatives/compare/refs/tags/v0.13.1...v0.13.2

Changelog: https://github.com/geigerzaehler/beets-alternatives/blob/v0.13.2/CHANGELOG.md

GaetanLepage committed:
aonsoku: init at 0.8.3 (#385245)

andresilva committed:
python3Packages.subliminal: set setuptools as build system

nixpkgs-merge-bot[bot] committed:
candy-icons: 0-unstable-2025-02-08 -> 0-unstable-2025-02-23 (#385762)

Co-authored-by: arunoruto <arunoruto@users.noreply.github.com>

GaetanLepage committed:
beetsPackages.alternatives: 0.13.1 -> 0.13.2 (#385784)

GaetanLepage committed:
ruff: 0.9.8 -> 0.9.9 (#385775)

r-ryantm committed:
epic5: 3.0.2 -> 3.0.3

nixpkgs-merge-bot[bot] committed:
epic5: 3.0.2 -> 3.0.3 (#385791)

Co-authored-by: Bot-wxt1221 <3264117476@qq.com>

usertam committed:
pcl: add usertam to maintainers

nixpkgs-merge-bot[bot] committed:
minio-warp: 1.1.0 -> 1.1.1 (#385625)

Co-authored-by: christoph-heiss <contact@christoph-heiss.at>

GaetanLepage committed:
pcl: fix build on darwin (#385701)

donovanglover committed:
goose-cli: 1.0.7 -> 1.0.10, add cloudripper as maintainer (#383788)

liberodark committed:
hot-resize: init at 0.1.1

Commit:
sedutil: 1.20.0 -> 1.49.6

rnhmjoj committed:
nixos/grub: generate BLS entries (#95901)

FliegendeWurst committed:
simple-completion-language-server: init at 0-unstable-2025-01-31 (#378338)

fabaff committed:
python312Packages.mypy-boto3-*: updates (#385667)

piotrkwiecinski committed:
n98-magerun2: 7.5.0 -> 8.0.0

nixpkgs-merge-bot[bot] committed:
awscli2: 2.23.11 -> 2.24.7 (#383267)

Co-authored-by: devusb <devusb@users.noreply.github.com>

GaetanLepage committed:
a-keys-path: init at 0.7.1 (#373994)

drupol committed:
n98-magerun2: 7.5.0 -> 8.0.0 (#385812)

r-ryantm committed:
vimPlugins.blink-cmp: 0.13.0 -> 0.13.1

nixpkgs-merge-bot[bot] committed:
tfswitch: 1.3.0 -> 1.4.0 (#385717)

Co-authored-by: psibi <psibi2000@gmail.com>

ulrikstrid committed:
lenovo-legion-app: 0.0.18 -> 0.0.19-unstable-2025-02-20 (#385594)

GaetanLepage committed:
coc-diagnostic: fix by removing dangling symlink

GaetanLepage committed:
vimPlugins.blink-cmp: 0.13.0 -> 0.13.1 (#385817)

SuperSandro2000 committed:
python313Packages.ipwhois: use libredirect.hook (#385608)

SuperSandro2000 committed:
opencv: cleanup unused code

SuperSandro2000 committed:
opencv: fix ade hash to fix gapi support

SuperSandro2000 committed:
opencv: fix fallback which causes infinite recursion

GaetanLepage committed:
coc-diagnostic: fix by removing dangling symlink (#385820)

erictapen committed:
weblate: 5.10.1 -> 5.10.2 (#385776)

GaetanLepage committed:
bumpp: init at 10.0.3 (#385728)

GaetanLepage committed:
taze: init at 18.6.0 (#385729)

pyrox0 committed:
sql-formatter: 15.4.10 -> 15.4.11 (#384288)

SuperSandro2000 committed:
paperless-ngx: set meta.mainProgram

Ma27 committed:
Merge: gitlab-ci-ls: 1.0.1 -> 1.0.2 (#385691)

r-ryantm committed:
dependency-track: 4.12.5 -> 4.12.6

r-ryantm committed:
all-the-package-names: 2.0.2042 -> 2.0.2070

r-ryantm committed:
ananicy-rules-cachyos: 0-unstable-2025-02-05 -> 0-unstable-2025-02-28

K900 committed:
paperless-ngx: set meta.mainProgram (#385841)

mweinelt committed:
opencv: fix ade hash to fix gapi support  (#385821)

dotlambda committed:
libdeltachat: 1.156.0 -> 1.156.1

Diff: https://github.com/deltachat/deltachat-core-rust/compare/refs/tags/v1.156.0...v1.156.1

Changelog: https://github.com/deltachat/deltachat-core-rust/blob/v1.156.1/CHANGELOG.md

donovanglover committed:
tray-tui: init at 0.1.0 (#385338)

nixpkgs-merge-bot[bot] committed:
amnezia-vpn: 4.8.3.2 -> 4.8.4.1 (#385779)

Co-authored-by: sund3RRR <sund3RRR@users.noreply.github.com>

donovanglover committed:
all-the-package-names: 2.0.2042 -> 2.0.2070 (#385840)

mweinelt committed:
openvino: enable npu/gpu support

CPU/NPU support only exists in Intel CPUs, but Intel GPUs can very well
be used on aarch64-linux platforms.

NPU previously failed to build, but after a recent opencv fix this became
possible.

Ma27 committed:
Merge: bind.dnsutils: remove reference to main output (#382577)

mweinelt committed:
openvino: enable npu/gpu support (#385853)

SuperSandro2000 committed:
webpack-cli: 5.1.4 -> 6.0.1 (#379373)

nixpkgs-merge-bot[bot] committed:
dapr-cli: 1.14.1 -> 1.15.0 (#385682)

Co-authored-by: JoshVanL <me@joshvanl.dev>

nixpkgs-merge-bot[bot] committed:
dependency-track: 4.12.5 -> 4.12.6 (#385839)

Co-authored-by: snue <kabelfrickler@gmail.com>

r-ryantm committed:
fedifetcher: 7.1.14 -> 7.1.15

stepbrobd committed:
goose-lang: 0.9.1 -> 0.9.2

patka-123 committed:
intelephense: 1.12.6 -> 1.14.1

also removed the update script because it will not magically update the lock file

SuperSandro2000 committed:
fedifetcher: 7.1.14 -> 7.1.15 (#385867)

SuperSandro2000 committed:
goose-lang: 0.9.1 -> 0.9.2 (#385866)

maralorn committed:
nix-output-monitor: 2.1.4 -> 2.1.5

SuperSandro2000 committed:
libdeltachat: 1.156.0 -> 1.156.1 (#385860)

r-ryantm committed:
oh-my-zsh: 2025-02-14 -> 2025-02-19 (#385869)

patka-123 committed:
vscode-extensions.bmewburn.vscode-intelephense-client: 1.12.5 -> 1.14.1

SuperSandro2000 committed:
morgen: 3.5.9 -> 3.6.6 (#367347)

GaetanLepage committed:
python312Packages.eliot: 1.16.0 -> 1.17.5

Diff: https://github.com/itamarst/eliot/compare/refs/tags/1.16.0...1.17.5

Changelog: https://github.com/itamarst/eliot/blob/1.17.5/docs/source/news.rst

nixpkgs-merge-bot[bot] committed:
spdx-license-list-data: 3.25.0 -> 3.26.0 (#369568)

Co-authored-by: oxzi <oxzi@users.noreply.github.com>

r-ryantm committed:
phpunit: 12.0.2 -> 12.0.5

marcusramberg committed:
k3s_1_29: 1.29.13 -> 1.29.14 (#385768)

risicle committed:
c-blosc2: 2.16.0 -> 2.17.0 (#385433)

prusnak committed:
python312Packages.pycuda: 2024.1.2 -> 2025.1 (#385387)

prusnak committed:
mxnet: don't try to build on darwin

prusnak committed:
mxnet: remove cuda support (#382891)

marcusramberg committed:
k3s_1_32: 1.32.1 -> 1.32.2 (#385771)

marcusramberg committed:
perlPackages.Tirex: fix build with recent mapnik (#366883)

marcusramberg committed:
perlPackages.SDL: fix build with GCC 14 (#370538)

r-ryantm committed:
python312Packages.weaviate-client: 4.10.4 -> 4.11.1

drupol committed:
phpunit: 12.0.2 -> 12.0.5 (#385893)

matteo-pacini committed:
dedup-darwin: init at 0.0.6

r-ryantm committed:
python312Packages.elevenlabs: 1.51.0 -> 1.52.0

marcusramberg committed:
k3s_1_30: 1.30.9 -> 1.30.10 (#385769)

wolfgangwalther committed:
maintainers: update fredeb (#382833)

marcusramberg committed:
k3s_1_31: 1.31.5 -> 1.31.6 (#385770)

SuperSandro2000 committed:
python313Packages.drf-extra-fields: add pytz dependency (#385255)

SuperSandro2000 committed:
python312Packages.eliot: 1.16.0 -> 1.17.5 (#385880)

prusnak committed:
lightning-terminal: init at 0.14.1-alpha (#383162)

SuperSandro2000 committed:
freefilesync: 14.0 -> 14.2 (#384269)

wolfgangwalther committed:
hubble: 0.13.6 -> 1.17.1 (#382313)

SuperSandro2000 committed:
python312Packages.flax: 0.10.3 -> 0.10.4 (#385676)

matteo-pacini committed:
vscode-extensions.equinusocio.vsc-material-theme-icons: remove at 3.8.8

matteo-pacini committed:
vscode-extensions.equinusocio.vsc-material-theme: remove at 34.7.5

dotlambda committed:
python312Packages.elevenlabs: 1.51.0 -> 1.52.0 (#385902)

NickCao committed:
prometheus-artifactory-exporter: 1.15.0 -> 1.15.1 (#385700)

NickCao committed:
infrastructure-agent: 1.60.1 -> 1.62.0 (#385692)

NickCao committed:
docker-credential-helpers: 0.8.2 -> 0.9.0 (#385685)

drupol committed:
vscode-extensions.equinusocio.vsc-material-theme*: remove (#385914)

r-ryantm committed:
hyprls: 0.5.0 -> 0.5.2

NickCao committed:
python312Packages.drf-yasg: 1.21.8 -> 1.21.9 (#385678)

drupol committed:
vscode-extensions.bmewburn.vscode-intelephense-client: 1.12.5 -> 1.14.1 (#385877)

drupol committed:
nix-output-monitor: 2.1.4 -> 2.1.5 (#385874)

NickCao committed:
python312Packages.google-cloud-dataproc: 5.17.0 -> 5.18.0 (#385677)

drupol committed:
intelephense: 1.12.6 -> 1.14.1 (#385873)

NickCao committed:
bosh-cli: 7.9.2 -> 7.9.3 (#385671)

NickCao committed:
python313Packages.aiolifx-themes: 0.6.7 -> 0.6.8 (#385665)

NickCao committed:
python313Packages.aiostreammagic: 2.10.0 -> 2.11.0 (#385664)

NickCao committed:
python313Packages.aiounifi: 81 -> 83 (#385663)

NickCao committed:
python313Packages.accuweather: 4.0.0 -> 4.1.0 (#385662)

NickCao committed:
python313Packages.aiohomeconnect: 0.13.0 -> 0.15.1 (#385661)

NickCao committed:
gqlgen: 0.17.65 -> 0.17.66 (#385751)

NickCao committed:
pulumi-bin: 3.152.0 -> 3.153.0 (#385744)

happysalada committed:
python312Packages.weaviate-client: 4.10.4 -> 4.11.1 (#385899)

NickCao committed:
python312Packages.google-cloud-dlp: 3.27.0 -> 3.28.0 (#385742)

SuperSandro2000 committed:
fwupd: use libredirect.hook (#385611)

GaetanLepage committed:
hot-resize: init at 0.1.1 (#385815)

NickCao committed:
stylance-cli: 0.5.4 -> 0.5.5 (#385716)

NickCao committed:
src-cli: 6.0.1 -> 6.1.0 (#385715)

nixpkgs-merge-bot[bot] committed:
hyprls: 0.5.0 -> 0.5.2 (#385923)

Co-authored-by: arthsmn <arthsmn@users.noreply.github.com>

NickCao committed:
python3Packages.subliminal: set setuptools as build system (#385786)

NickCao committed:
pretender: 1.3.1 -> 1.3.2 (#385781)

NickCao committed:
python312Packages.python-roborock: 2.11.2 -> 2.12.0 (#385780)

NickCao committed:
python312Packages.publicsuffixlist: 1.0.2.20250227 -> 1.0.2.20250228 (#385774)

SuperSandro2000 committed:
dedup-darwin: init at 0.0.6 (#385897)

NickCao committed:
python312Packages.lm-format-enforcer: 0.10.9 -> 0.10.11 (#385766)

NickCao committed:
summon: 0.10.2 -> 0.10.3 (#385765)

NickCao committed:
msolve: 0.7.3 -> 0.7.4 (#385758)

b-rodrigues committed:
rPackages.prqlr: fixed build
